transport/src/main/java/io/netty/channel/DefaultChannelPipeline.java

```diff
@@ -83,6 +85,23 @@
      */
     private boolean registered;
 
+    private static final AtomicIntegerFieldUpdater<DefaultChannelPipeline> ENQUEUE_WAKEUP_TASK_UPDATER;
+
+    static {
+        AtomicIntegerFieldUpdater<DefaultChannelPipeline> enqueueWakeupTaskUpdater =
+                PlatformDependent.newAtomicIntegerFieldUpdater(DefaultChannelPipeline.class, "enqueueWakeupTask");
+        if (enqueueWakeupTaskUpdater == null) {
+            enqueueWakeupTaskUpdater = AtomicIntegerFieldUpdater.newUpdater(DefaultChannelPipeline.class,
+                                                                            "enqueueWakeupTask");
+        }
+        ENQUEUE_WAKEUP_TASK_UPDATER = enqueueWakeupTaskUpdater;
+    }
+
+    private volatile int enqueueWakeupTask;
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this unused "enqueueWakeupTask" private field. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://garage.netty.io/sonarqube/coding_rules#rule_key=squid%3AS1068)